### PR TITLE
home-work5

### PR DESCRIPTION
**Pull request by AGB1Q84**
Lsn-5

### CODE REVIEW

File: Maksatova Benazir/JavaScript/LSN-3/home work/main.js [hunk 1]
```diff
@@ -1,68 +1,68 @@
...
+console.log(result);
 
 // 7 exercise
-// let cycle7 = { green: "зеленый", red: "красный", blue: "голубой" };
-// for (let key in cycle7) {
-//   if (cycle7.hasOwnProperty(key)) {
-//     console.log(`Ключ: ${key}, Элемент: ${cycle7[key]}`);
-//   }
-// }
+let cycle7 = { green: "зеленый", red: "красный", blue: "голубой" };
```
Comment (bayzena) [inline Maksatova Benazir/JavaScript/LSN-3/home work/main.js:44]: Observe tabs

File: Maksatova Benazir/JavaScript/LSN-3/home work/main.js [hunk 2]
```diff
@@ -1,68 +1,68 @@
...
+for (let key in cycle7) {
+  if (cycle7.hasOwnProperty(key)) {
```
Comment (bayzena) [inline Maksatova Benazir/JavaScript/LSN-3/home work/main.js:46]: Why this check? Do you go through the keys of this object and check whether it is in the same object?

File: Maksatova Benazir/JavaScript/LSN-4/home work/script.js [hunk 3]
```diff
@@ -28,4 +28,4 @@ let result = someFn4(f5, f6, f7);
 function someFn4(f5, f6, f7) {
   return (f5 - f6) / f7;
 }
-console.log(someFn4);
+console.log(result);
```
Comment (bayzena) [inline Maksatova Benazir/JavaScript/LSN-4/home work/script.js:31]: Where is result from?

File: Maksatova Benazir/JavaScript/LSN-5/home work/script.js [hunk 4]
```diff
@@ -0,0 +1,145 @@
+// Object
+// 1 exercise
+let user = {
+  a: { b: 3 },
```
Comment (bayzena) [inline Maksatova Benazir/JavaScript/LSN-5/home work/script.js:4]: Observe tabs

File: Maksatova Benazir/JavaScript/LSN-5/home work/script.js [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+let user3 = {
+  key1: 0,
+  key2: 1,
+  key3: 2,
+  key4: 3,
+  key5: 4,
+  key6: 5,
+  key7: 6,
+  key8: 7,
+  key9: 8,
```
Comment (bayzena) [inline Maksatova Benazir/JavaScript/LSN-5/home work/script.js:29]: key1: 0,
  key2: 1,
  key3: 2,
  key4: 3,
  key5: 4,
  key6: 5,
  key7: 6,
  key8: 7,
  key9: 8,
  
  
No need, you rewrite them
const user3={}

**Review by bayzena — CHANGES_REQUESTED** [review]

**Comment by AGB1Q84** [comment]
Домашняя работа

**Pull request merged**